packages/presentation/src/components/Members.svelte

```diff
@@ -15,7 +15,7 @@
 <script lang="ts">
   import contact, { Employee, EmployeeAccount } from '@hcengineering/contact'
   import { Account, DocumentQuery, Ref, SortingOrder, Space } from '@hcengineering/core'
-  import login from '@hcengineering/login'
+  import workbench from '@hcengineering/workbench'
```

Comment:
@BykhovDenis @ThetaDR I suppose presentation should not know about any of login or even workspace, even for interfaces.

Reply:
Yes, it shouldn't. But it was implemented here, I think we should move it to another place in a separate PR

Reply:
ok